draft-ietf-jsonpath-base.md

```diff
@@ -2212,6 +2225,20 @@ The resulting class of attacks, *injections* (e.g., SQL injections),
 is consistently found among the top causes of application security
 vulnerabilities and requires particular attention.
 
+## Attack Vectors on the Data Examined by JSONPath Queries {#data-attack}
```

Comment:
Really?  I'm having trouble believing this.  I think you should lose this and stick with the good wording up at the top of the review.  This is really extraneous to Murray's comment.

Reply:
Well, it was the main reason *I* care for the clear separation between query and data errors.
So I don't think any explanation of this is complete without this.
But I agree we don't *have* to explain all that.

Reply:
Murray asked for a little bit of explanation for a basic fact about JSONPath that frequently surprises people.  I think adding a new security consideration is overkill, and also it's a significant change that I think the WG should get to comment on.  So I still think we should just skip this.

Reply:
I split out the seccons part to #493, so #492 should now be clean of security